### DISCUSSION TITLE
Welcome to material-sample Discussions!

### DISCUSSION THREAD

**Discussion by stanblum** [tdwg/material-sample#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dagendresen** [comment]
My name is Dag Endresen and I am the GBIF node manager for Norway. My previous engagement in TDWG dates back to my first TDWG conference in 2004 in New Zealand when my primary interest was in using and expanding TDWG standards to facilitate the publication of agrobiodiversity seedbank/genebank data (including crop breeding target traits) in GBIF from my role as GBIF node manager for the Nordic Gene Bank. Before joining the Norwegian GBIF node I was briefly the Knowledge System Engineer at the GBIF Secretariat including close interaction with many related TDWG processes. My current primary interest in the MaterialSample Discussions focuses on expanding TDWG standards for improved publication of museum collection specimens in GBIF. Many GBIF data publishers in Norway are also deeply involved in mobilizing DNA-sequence-based data from eDNA and other sources in GBIF, which is my second interest in the MaterialSample process.

**Reply by stanblum** [reply]
Hi Dag!  Thanks for posting your introduction.  With your long experience in the seedbank/germplasm community, how do people think of their holdings, as natural occurrences of organisms (i.e., where taxonomy, time, and place are primarily important) or as resources in collections where taxonomy and traits are most important because cultivation makes "where" less important?  Or is this not a useful distinction?

**Reply by dagendresen** [reply]
I do believe that seedbank/genebank people think of their holdings (accessions) as collection items (specimens, some type of dwc:LivingSpecimen). Even if the seeds remain alive they would be stored in cold storage on shelves in jars or in small aluminum bags for many decades. However, from time to time some of the frozen seed samples are tawed and planted out in field (or greenhouse) experiments - or included as gene donors in breeding programs - (where the location and environmental conditions matter a lot). And, after discussing with Rich Pyle @deepreef, I find it very, very fascinating to think of the seedbank specimens (accessions) in this stage as a sort of a dwc:Organism taking part in a (in vivo and probably not in situ) dwc:Occurrence. Does it make sense that seedbank samples could pass in this manner from dwc:LivingSpecimen to dwc:Organism and back to dwc:LivingSpecimen again?
